### ISSUE TITLE
Hyperparameters and Error Metric

### ISSUE THREAD

**Issue by vgoklani** [Vict0rSch/deep_learning#9]
Just a few basic questions:
1. Why did you choose this particular network structure? Since you have two LSTM layers, would you get better performance from using the "relu" activation function? 
2. Why are you only running 1 epoch? Is that just for testing purposes?
3. How are you measuring errors for forecasting, what is your chosen error metric? Most other results use RMSE, however, is there a better metric? RMSE feels like a necessary but not sufficient condition for measuring the "learning quality" of a neural net.
